autoware_adapi_v1_msgs/vehicle/msg/GeoPointStamped.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
+float64 latitude
+float64 longitude
```

Comment:
This was inspired by https://github.com/ros-geographic-info/geographic_info/blob/ros2/geographic_msgs/msg/GeoPoint.msg I assume.

Why not use `ros-humble-geometry-msgs` directly here? They have both `GeoPoint.msg` and `GeoPointStamped.msg`

Reply:
I change to use geographic_msgs `GeoPointStamped.msg` at 6ea163c